plan/data.go

```diff
@@ -13,5 +13,5 @@ func RepositoryCommitDataFilename(emptyData interface{}) string {
 }
 
 func SourceUnitDataFilename(emptyData interface{}, u *unit.SourceUnit) string {
-	return filepath.Clean(fmt.Sprintf("%s/%s.%s", u.Name, u.Type, buildstore.DataTypeSuffix(emptyData)))
+	return filepath.Clean(fmt.Sprintf("%s%s%s.%s", u.Name, string(filepath.Separator), u.Type, buildstore.DataTypeSuffix(emptyData)))
```

Comment:
At this point, the value of using `fmt.Sprintf` is getting pretty low. If you want, you could rewrite this using string concatenation instead.

Either way is fine though.

Or maybe it could be `filepath.Join(u.Name, u.Type+"."+buildstore.DataTypeSuffix(emptyData))`.